amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/avro/parser/domain/AvroEnumParser.scala

```diff
@@ -0,0 +1,34 @@
+package amf.apicontract.internal.spec.avro.parser.domain
+
+import amf.apicontract.internal.spec.avro.parser.context.AvroSchemaContext
+import amf.core.client.scala.model.domain.ScalarNode
+import amf.core.client.scala.vocabulary.Namespace.XsdTypes
+import amf.core.internal.parser.{YMapOps, YScalarYRead}
+import amf.shapes.client.scala.model.domain.AnyShape
+import amf.shapes.internal.domain.metamodel.AnyShapeModel
+import org.yaml.model._
+
+class AvroEnumParser(map: YMap)(implicit ctx: AvroSchemaContext) extends AvroScalarShapeParser("string", Some(map)) {
+
+  override def parse(): AnyShape = {
+    val shape = super.parse()
+    map.key("name", (AnyShapeModel.Name in shape).allowingAnnotations)
```

Comment:
"name", "namespace", "aliases" and "doc" are parsed in multiple shapes (Enum, Record, Fixed) the same way. They could be extracted to a trait or similar.

Reply:
done